pandas/tests/indexes/test_multi.py

```diff
@@ -1425,8 +1448,6 @@ def test_equals_multi(self):
 
         self.assertFalse(self.index.equals(self.index[:-1]))
 
-        self.assertTrue(self.index.equals(self.index._tuple_index))
```

Comment:
added test. And they *don't* evaluate as equals (nor did they in master). the are equivalent, but not equal (you have to wrap it in a MI to make it equal, NOT an ``Index``)